### PR TITLE
Added shortcuts to markdown mode

### LINKED ISSUE #216: Markdown editor needs to support some keyboard shortcuts
Examples
`cmd + b` - `**text**`
`cmd + i` - `_text_`

### PR DESCRIPTION
**Pull request by AbhayVAshokan**
Fixes #216 
- Added shortcuts to markdown mode.

Demo: https://vimeo.com/703680106/be26a0367e

@labeebklatif _a please review.

### CODE REVIEW

File: lib/components/Editor/CustomExtensions/Markdown/helpers.js [hunk 1]
```diff
@@ -1,3 +1,124 @@
+const applyMarkdownWordFormatting = (inputFieldRef, decorator = "") => {
```
Comment (labeebklatif) [inline lib/components/Editor/CustomExtensions/Markdown/helpers.js:1]:
```suggestion
const toggleMarkdownWordFormatting = (inputFieldRef, decorator = "") => {
```

File: lib/components/Editor/CustomExtensions/Markdown/helpers.js [hunk 2]
```diff
@@ -1,3 +1,124 @@
...
+  const selectionEnd = textArea.selectionEnd;
+  const decoratorLength = decorator.length;
+  let front = textArea.value.substring(0, selectionStart);
+  let middle = textArea.value.substring(selectionStart, selectionEnd);
+  let back = textArea.value.substring(selectionEnd, textArea.value.length);
+
+  if (selectionStart === selectionEnd) return;
+
+  if (
+    front.substr(-decoratorLength) == decorator &&
```
Comment (labeebklatif) [inline lib/components/Editor/CustomExtensions/Markdown/helpers.js:16]:
```suggestion
    front.endsWith(decorator) &&
```

File: lib/components/Editor/CustomExtensions/Markdown/helpers.js [hunk 3]
```diff
@@ -1,3 +1,124 @@
...
+    back.substr(0, decoratorLength) == decorator
```
Comment (labeebklatif) [inline lib/components/Editor/CustomExtensions/Markdown/helpers.js:17]:
```suggestion
    back.startsWith(decorator)
```

File: lib/components/Editor/CustomExtensions/Markdown/helpers.js [hunk 4]
```diff
@@ -1,3 +1,124 @@
...
+  ) {
+    decorator = "";
+    front = front.substring(0, front.length - decoratorLength);
+    back = back.substring(decoratorLength, back.length);
+  } else if (
+    middle.substr(0, decoratorLength) == decorator &&
```
Comment (labeebklatif) [inline lib/components/Editor/CustomExtensions/Markdown/helpers.js:23]: Let's use `String.startsWith` and `String.endsWith` wherever possible

**Review by labeebklatif — COMMENTED** [review]
@AbhayVAshokan _a Please have a look at the comments

**Comment by AbhayVAshokan** [comment]
@labeebklatif _a all the requested changes have been done.

**Review by labeebklatif — APPROVED** [review]

**Pull request merged**